### PR TITLE
Database

### PR DESCRIPTION
**Pull request by annelau99**

### CODE REVIEW

File: server/routes/groupRoutes.js [hunk 1]
```diff
@@ -139,5 +139,104 @@ router.post('/:groupName/change/:key', (req, res) => {
   });
 });
 
+
+//delete a group
+//TODO: make sure only admin of group can delete a group
+router.post('/:groupName/delete', (req, res) => {
+  //req.user.username --> check if this is admin
```
Comment (annelau99) [inline server/routes/groupRoutes.js]: does this comment need to be left in

File: server/routes/groupRoutes.js [hunk 2]
```diff
@@ -139,5 +139,104 @@ router.post('/:groupName/change/:key', (req, res) => {
...
+    return;
+  });
+});
+
+//add a user to a group
+//TODO: make sure only admin of group can add user to a group
+router.post('/:groupName/add', (req, res) => {
+  let groupName = req.params.groupName;
+  let username = req.body.username;
+  let group = db.Group.findByPk(req.params.groupName).then((modelInstance) => {
```
Comment (annelau99) [inline server/routes/groupRoutes.js]: is this async?

Comment (waymao) [reply server/routes/groupRoutes.js]: I agree that if this function is async, `let group = ` should be removed?

Comment (annelau99) [reply server/routes/groupRoutes.js]: idk if it is async!

**Pull request merged**